deploy/chart/templates/csi-rclone-storageclass.yaml

```diff
@@ -6,9 +6,10 @@ metadata:
   {{- include "chart.labels" . | nindent 4 }}
```

Comment:
The name of the storage class should be hardcoded or even better set in a values file with a default value. It should not be templated from the helm chart name.

Reply:
TODO in a followup PR.